src/domains/transaction/components/TransactionDetail/TransactionResponsiveIcon/TransactionResponsiveIcon.tsx

```diff
@@ -24,5 +24,5 @@ export const TransactionResponsiveIcon = ({ icon }: { icon: string }) => {
 	);
 };
 
-export const TransactionDelegateIcon = () => <TransactionResponsiveIcon icon="Delegate" />;
-export const TransactionDelegateResignationIcon = () => <TransactionResponsiveIcon icon="DelegateResignation" />;
+export const TransactionDelegateIcon = () => <TransactionResponsiveIcon icon="UsernameRegistration" />;
```

Comment:
this changes the delegate icon, that's not what we want as delegate registrations are still used for ARK transactions + this affects sender/recipients when they are a delegate
<img width="553" alt="image" src="https://github.com/ArdentHQ/arkvault/assets/35610748/9b9b5277-f2d8-485c-adb5-fb31d95ca7f6">